src/routes/CreateSafePage/steps/OwnersAndConfirmationsNewSafeStep.tsx

```diff
@@ -142,6 +142,7 @@ function OwnersAndConfirmationsNewSafeStep(): ReactElement {
                         createSafeForm.change(nameFieldName, addressName)
                       }
                     }}
+                    value={createSafeFormValues[addressFieldName]}
```

Comment:
Why are you passing a value here? Cannot all the logic of prepending the `shortName` occur inside the component?